cmake/projects/NASM/schemes/url_sha1_nasm_unpack_windows.cmake.in

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2017, Zhuhao Wang
```

Comment:
> Line 47

Oh... Indeed, sorry :)

I think it make sense to rename this scheme to `url_sha1_unpack_bin_install` and move it to directory with global schemes `cmake/schemes`.

Also we need to handle this scheme here:
* https://github.com/ruslo/hunter/blob/4072b31487e0a9b072e38e2616375144d6071b00/cmake/modules/hunter_pick_scheme.cmake#L74

And update documentation file:
* https://github.com/ruslo/hunter/blob/4072b31487e0a9b072e38e2616375144d6071b00/docs/reference/internal-variables/hunter_package_scheme.rst